plugins/event_source/aws_cloudtrail.py

```diff
@@ -37,6 +38,8 @@
 
 from aiobotocore.session import get_session
 
+from .aws_sqs_queue import connection_args
```

Comment:
This creates an interdependence between plugins which I think is a bad idea. What do you think moving it to a new file such as aws_utils.py ?

Reply:
I think it is better to do it now, no reason to wait for another aws plugin